constants/api.js

```diff
@@ -1,35 +1,20 @@
-import qs from 'qs';
+import qs from 'querystring';
 
 const API_URL = '/api';
-const AUTH_URL = '/auth';
-
-const queryOptions = { addQueryPrefix: true };
 
 export default {
-  auth: {
-    login: `${AUTH_URL}/login`,
-    register: `${AUTH_URL}/register`,
-    logout: `${AUTH_URL}/logout`,
-  },
   storage: {
     getMainPage: `${API_URL}/storage/main-page`,
     getSidebar: `${API_URL}/storage/sidebar`,
   },
   core: {
-    uploads: `${API_URL}/core/uploads`,
```

Comment:
did it work before? :)

Reply:
should upload to cloudinary:
https://github.com/babajka/babajka-backend/blob/dfd78f437999d7f4fc0078a7a2eb1ed5c9fe1c40/src/api/core/controller.js#L12